lib/ansible/plugins/action/nxos_file_copy.py

```diff
@@ -213,6 +213,7 @@ def file_push(self):
 
         remote_file = remote_file or os.path.basename(local_file)
         remote_exists = self.remote_file_exists(remote_file, file_system)
+        self.results['changed'] = False
```

Comment:
@trishnaguha What is the reason for this change?

Reply:
@mikewiebe If there is no change made to the device the results dictionary should have the key `changed` set to False. We have integration tests failing for this https://github.com/ansible/ansible/blob/devel/test/integration/targets/nxos_file_copy/tests/cli/negative.yaml#L54
This is applicable both for file pull and file push.